config/checkstyle/checkstyle.xml

```
@@ -155,6 +155,9 @@
     LENGTH and CODING CHECKS
```

Commit message:

> Nonexplanatory empty catch blocks are left unchecked.

It's not clear what is meant by `Nonexplanatory empty catch blocks`.